Apps/Sandcastle/gallery/Custom Geocoder.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Cluster labels, billboards and points.">
+    <meta name="cesium-sandcastle-labels" content="Tutorials,Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        require.config({
+            baseUrl : '../../../Source',
+            waitSeconds : 60
+        });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #toolbar {
+        background: rgba(42, 42, 42, 0.8);
+        padding: 4px;
+        border-radius: 4px;
+    }
+    #toolbar input {
+        vertical-align: middle;
+        padding-top: 2px;
+        padding-bottom: 2px;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var options = {
+    geocoder: {
+        getSuggestions: function () {
+            return [];
+        },
+        geocode: function (input, callback) {
+            var endpoint = 'http://nominatim.openstreetmap.org/search?';
+            var query = 'format=json&q=' + input;
+            var requestString = endpoint + query;
+            Cesium.loadJson(requestString)
+                    .then(function (results) {
+                        var bboxDegrees;
+                        callback(undefined, results.map(function (resultObject) {
+                            bboxDegrees = resultObject.boundingbox;
+                            return {
+                                displayName: resultObject.display_name,
+                                bbox: {
```

Comment:
`bbox` is part of the new public Cesium API, right?  This is part of the "class" a geocode implementation returns.

It should following Cesium conventions:
* Avoid abbreviations, probably call it `rectangle`
* Use a Cesium type, probably `Rectangle`
* Use radians.  This will likely require most users to convert, e.g., using `fromDegrees`, but it will be consistent with the rest of the API.